guidelines.md

```diff
@@ -42,21 +42,21 @@ Examples are not:
 ### Throughout
 
 * Should point to documentation on first mention:
-  [kubectl](../docs/user-guide/kubectl-overview.md),
-  [pods](../docs/user-guide/pods.md),
-  [services](../docs/user-guide/services.md),
-  [deployments](../docs/user-guide/deployments.md),
-  [replication controllers](../docs/user-guide/replication-controller.md),
-  [jobs](../docs/user-guide/jobs.md),
-  [labels](../docs/user-guide/labels.md),
-  [persistent volumes](../docs/user-guide/persistent-volumes.md),
+  [kubectl](https://github.com/kubernetes/kubernetes/tree/master/docs/user-guide/kubectl-overview.md),
```

Comment:
unfortunately this is no longer where the docs live. Can you please follow the redirections and link directly to kubernetes.io website?